src/_locales/ru/messages.json

```
@@ -134,9 +134,9 @@
     "html_popup_suspend_selected_tabs": { "message": "Приостановить выбранные вкладки" },
     "html_popup_unsuspend_selected_tabs": { "message": "Возобновить выбранные вкладки" },
     "html_popup_settings": { "message": "Настройки" },
-    "html_restoring_title": { "message": "Restoring window" },
-    "html_restoring_heading": { "message": "Restoring tabs" },
-    "html_restoring_detail": { "message": "Please wait while I restore this window..." },
+    "html_restoring_title": { "message": "Восстановление окна" },
+    "html_restoring_heading": { "message": "Восстановление вкладок" },
+    "html_restoring_detail": { "message": "Пожалуйста, подождите пока я восстановлю это окно..." },
```

```suggestion
    "html_restoring_detail": { "message": "Пожалуйста, подождите, пока я восстановлю это окно..." },
```

(complicated sentence: "[you] wait" + "I restore", have to be separated by comma)